core/src/test/scala/org/elasticmq/actor/QueueActorMsgOpsTest.scala

```diff
@@ -147,7 +147,7 @@ class QueueActorMsgOpsTest extends ActorTest with QueueManagerForEachTest with D
     val q1 = createQueueData("q1", MillisVisibilityTimeout(1L))
 
     for {
-      Right(queueActor) <- queueManagerActor ? CreateQueue(q1)
+      Right(queueActor) <- queueManagerActor ? CreateQueue(q1.copy(maxReceiveCount = 2))
```

Comment:
I think you are looking at the docs of AWS CloudFormation, not SQS - maybe there's a typo in one of the CloudFormation docs. In the SQS docs (e.g. http://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_CreateQueue.html), the `RedrivePolicy` is specified as *optional* (which it must be, to maintain backwards-compatibility).

This is also visible in the web interface:
http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-dead-letter-queues.html